<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>535</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>635</width>
    <height>535</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>635</width>
    <height>535</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>智能大棚---主界面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resource.qrc">
    <normaloff>:/png/ui_icon.ico</normaloff>:/png/ui_icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(86, 118, 162);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>监控台</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ui_refresh">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>更新</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollArea" name="ui_scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="ui_scroll_area">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>613</width>
          <height>473</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="ui_layout">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="0" column="2">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="monitor_1">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>200</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="label_1">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="monitor_2">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>200</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="label_2">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="monitor_3">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>200</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="label_3">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="monitor_4">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>200</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="label_4">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
